### PR TITLE
fix plugins directories

### PR DESCRIPTION
**Pull request by sabban**
fix plugins directories

### CODE REVIEW

File: wizard.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ CROWDSEC_PATH="/etc/crowdsec"
 CROWDSEC_CONFIG_PATH="${CROWDSEC_PATH}"
 CROWDSEC_LOG_FILE="/var/log/crowdsec.log"
 LAPI_LOG_FILE="/var/log/crowdsec_api.log"
-CROWDSEC_PLUGIN_DIR="/var/lib/crowdsec/plugins/"
+CROWDSEC_PLUGIN_DIR="${CROWDSEC_USR_DIR}/plugins"
```
Comment (blotus) [inline wizard.sh:26]: We will need to update the wizard patches to also change `${CROWDSEC_USR_DIR}`

File: rpm/SPECS/crowdsec.spec [hunk 2]
```diff
@@ -39,6 +39,7 @@ BuildRequires:  systemd
 
 %build
 BUILD_VERSION=%{local_version} make
+sed -i "s#/usr/local/lib/crowdsec/plugins/#/usr/lib/crowdsec/plugins/#g" config/config.yaml
```
Comment (blotus) [inline rpm/SPECS/crowdsec.spec:42]: Why sed and not a patch ?

File: wizard.sh (hunk 1 above)
Comment (sabban) [reply wizard.sh:26]: this should be ok, because we don't need it when packaging

File: rpm/SPECS/crowdsec.spec (hunk 2 above)
Comment (sabban) [reply rpm/SPECS/crowdsec.spec]: because it can be either `/usr/lib` or `/usr/lib64`. But still, this needs using macros...

**Review by blotus — APPROVED** [review]

**Pull request merged**